.github/workflows/build-go.yml

```
@@ -298,11 +298,7 @@ jobs:
     - name: Run tests
       run: gotestsum --no-color=false -- -race -coverprofile=cover.out -covermode=atomic -vet=off ./...
 
-    - name: Quality Gate
+    - name: Test Coverage
       run: |
         total=`go tool cover -func=cover.out | grep total | grep -Eo '[0-9]+\.[0-9]+'`
         echo "Total test coverage = $total%"
```

If this is just informative, don't you think we could just run it and keep the whole output ?
```suggestion
       go tool cover -func=cover.out
```

---

We could